src/config/dynamic-theme-fixes.config

```diff
@@ -538,6 +538,7 @@ allegro.pl
 
 INVERT
 i[title="Smart!"]
+div[class="mpof_ki m389_6m"]
```

Comment:
Seems like random classes to?

Reply:
Whole code is one big mess. This hack just works. For now.